vilebot/src/main/java/com/oldterns/vilebot/handlers/user/Karma.java

```diff
@@ -165,6 +171,70 @@ private void karmaDec( ReceivePrivmsg event )
         }
     }
 
+    @Handler
+    public void karmaIncOrDec( ReceivePrivmsg event )
+    {
+        // Match any string that has at least one word that ends with ++
+        Matcher incOrDecMatcher = incOrDecPattern.matcher( event.getText() );
+
+        if ( incOrDecMatcher.matches() )
+        {
+            if ( isPrivate( event ) )
+            {
+                decideIncOrDec( event, event.getSender() );
```

Comment:
We probably don't want people do PM this ability, no? We could do what the inc/dec does (remove karma from the sender instead) or just have the same behaviour as increasing karma outside of the foobar.